recipes/shapelysmooth/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "shapelysmooth" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/shapelysmooth-{{ version }}.tar.gz
+  sha256: 02141a11f8f615b418a854475292c107b3e2c6bc5f56e9b683eea9f63ac288a3
+
+build:
+  skip: True  # [win or osx]
```

Comment:
Upstream has wheels for Windows and macOS, can we not get those to work here by adjusting some library dependencies below?

Reply:
Tried a little whack-a-mole in the commits above but couldn't find the libraries added here in Mac or windows